src/activities/ActivityUsageEntity.js

```diff
@@ -104,4 +104,22 @@ export class ActivityUsageEntity extends Entity {
 		const fields = [{ name: 'dueDate', value: dateString }];
 		await performSirenAction(this._token, action, fields);
 	}
+
+	canEditDraft() {
+		return this._entity && this._entity.hasActionByName(Actions.activities.updateDraft);
+	}
+
+	async setDraftStatus(isDraft) {
+		if (!this.canEditDraft()) {
+			return;
+		}
+
+		let updateDraftAction = this._entity.getActionByName(Actions.activities.updateDraft);
+		const fields = [{ name: Classes.activities.draft, value: isDraft }];
```

Comment:
I think this is better to just leave as the string `'draft'`, although I don't have a strong preference - mostly, I see this as "the class happens to be the same as the field name", not that they are necessarily the same.

That said, we also haven't really library-ified field names at all; no real reason for that as far as I know, just hasn't ever been done ☹️